### PR TITLE
Documentation for new Remote Assets feature.

### PR DESCRIPTION
**Pull request by HitmanInWis**
Thumbnail to be added tomorrow hopefully, but want to get this up for review since it is rather extensive.

### CODE REVIEW

File: _acs-aem-commons/features/on-deploy-scripts/index.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: On-Deploy Scripts
 description: Create one-time scripts that execute upon deployment to AEM
 date: 2018-04-05
 redirect_from: /acs-aem-commons/features/on-deploy-scripts.html 
-feature-tags: new administration backend-dev content-migration
```
Comment (HitmanInWis) [inline _acs-aem-commons/features/on-deploy-scripts/index.md]: Since this feature is no longer `new` I removed the tag :)

**Comment by HitmanInWis** [comment]
This is the associated documentation for https://github.com/Adobe-Consulting-Services/acs-aem-commons/pull/1707

File: _includes/shared/feature-card.html [hunk 2]
```diff
@@ -6,8 +6,8 @@
 {% elsif feature.last-updated-release == project.version or feature.last-updated-release == project.version60 %} updated {% endif %}">
     <a class="feature-image-wrapper" href="{{ feature.url }}">
 
-        {% if feature.tags contains 'deprecated' %}<div class="deprecated"></div>
-        {% elsif feature.tags contains 'beta' %}<div class="beta"></div>
+        {% if feature.feature-tags contains 'deprecated' %}<div class="deprecated"></div>
+        {% elsif feature.feature-tags contains 'beta' %}<div class="beta"></div>
```
Comment (HitmanInWis) [inline _includes/shared/feature-card.html]: Fixed a bug here where these banners were not working b/c the wrong var name was being referenced.

**Comment by HitmanInWis** [comment]
Thumbnail added, this is ready for review.

**Comment by HitmanInWis** [comment]
awesome, thanks @davidjgonzalez !

**Comment by HitmanInWis** [comment]
@justinedelson do we anticipate merging this soon, now that 4.1.0 has been released for a couple weeks?

**Comment by badvision** [comment]
Sorry!  This slipped under my radar.

**Pull request merged**